src/account/operations/balance.rs

```diff
@@ -190,11 +223,15 @@ impl AccountHandle {
                 }
             }
         }
+
         log::debug!(
             "[BALANCE] total_amount: {}, locked balance: {}",
             total_amount,
             locked_amount
         );
+
+        // TODO should we add total_rent_amount to locked_amount here ?
```

Comment:
Yes I think that's what we said we want to do

Reply:
Wait shouldn't it be a minus actually ? `total_amount < locked_amount - total_rent_amount`

Reply:
My comment was about L 275 `available: total_amount - locked_amount - total_rent_amount,` where the minus is doing the same as adding it to the locked amount before
And in this check it needs to be + then, since we subtract both at the end, no?
Or just do `locked_amount += total_rent_amount;` before both things?

Reply:
Yeah sounds good, did that